full-service/src/json_rpc/v1/models/receiver_receipt.rs

```diff
@@ -95,7 +95,7 @@ mod tests {
         let account_key = AccountKey::random(&mut rng);
         let public_address = account_key.default_subaddress();
         let txo = TxOut::new(
-            BlockVersion::MAX,
+            BlockVersion::TWO,
```

Comment:
Why was this changed? Seems valuable to have a test that tests the newest block version.
In other places we put such tests in a loop that tries all relevant block versions.

Reply:
yeah agreed, was testing to see what would change. I'll put TODO comments from now on, bad habit of mine not to.

Reply:
Will create a gh issue to track that

Reply:
What breaks if it is set to ::MAX?

Reply:
Nothing now, it should be back to MAX

Reply:
oh wait I missed this one, hahha